docs/pages/build-reference/easignore.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Ignoring files
+description: Learn how to configure EAS to ignore unnecessary files during the build process.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+
+A `.easignore` file defines which files [EAS](https://expo.dev/eas) should ignore when uploading your project to the [EAS Build](/build/introduction) servers.
+
+> **info** Ignoring unnecessary files can help reduce your app's archive size and upload time.
+
+By default, the [EAS CLI](/build/setup/#install-the-latest-eas-cli) refers to the [`.gitignore`](https://git-scm.com/docs/gitignore) file (if it exists) to determine which files to ignore. If you create a `.easignore` file, the EAS CLI prioritizes it over the `.gitignore` file. When creating a `.easignore` file, include all files from your `.gitignore` file, and add additional files you want to ignore.
+
+## Getting started with .easignore
+
+<Step label="1">
+
+Create a `.easignore` file in the root of your project.
+
+</Step>
+
+<Step label="2">
+
+Copy the content of the `.gitignore` file into the `.easignore` file. Then, add any files that are unnecessary for the build process.
+
+```bash .easignore
+# Copy everything from your .gitignore file here
+
+# Additional ignores specific to EAS builds
+
+# Ignore native folders (If you are using CNG)
+/ios
+/android
+
+# Ignore test coverage reports
+/coverage
+```
+
+Learn more about [Continuous Native Generation (CNG)](https://docs.expo.dev/workflow/continuous-native-generation).
```

Comment:
```suggestion
Learn more about [Continuous Native Generation (CNG)](/workflow/continuous-native-generation).
```

I think we should add some context on why someone should learn at CNG. We can briefly explain here the comment from the above snippet on why to add android and ios directories to this file and the link to the CNG guide in the context.